lib/common/widgets/link_tile.dart

```diff
@@ -31,10 +34,13 @@ class LinkTile extends StatelessWidget {
         DiscoverType.User => Routes.user(id, info),
         DiscoverType.Review => Routes.review(id, info),
       }),
-      onLongPress: () {
-        if (discoverType == DiscoverType.Anime ||
-            discoverType == DiscoverType.Manga) {
-          showSheet(context, EditView((id: id, setComplete: false)));
+      onLongPress: () async {
+        switch (discoverType) {
+          case DiscoverType.Anime || DiscoverType.Manga:
+            showSheet(context, EditView((id: id, setComplete: false)));
+          case DiscoverType.User:
+            await Clipboard.setData(ClipboardData(text: "$additionalData"));
```

Comment:
For copying to clipboard, I use the `Toast.copy` helper function I made in `common/widgets/overlays/toast.dart`.
Why is that file in the `widgets` directory? No idea, I'm just noticing, I will move it out of there soon.